editor/editor_properties.cpp

```diff
@@ -2849,10 +2892,17 @@ EditorProperty *EditorInspectorDefaultPlugin::get_editor_for_property(Object *p_
 						if (slice == "or_lesser") {
 							lesser = true;
 						}
+						if (slice.begins_with("suffix:")) {
+							suffix = " " + slice.replace_first("suffix:", "").strip_edges();
+						}
 					}
 				}
 
-				editor->setup(min, max, step, hide_slider, exp_range, greater, lesser);
+				if (angle && suffix == String()) {
+					suffix = String::utf8("°");
```

Comment:
I would use something like this, to avoid issues if compiler uses wrong file encoding.
```suggestion
					suffix = U"\u00B0";
```

Reply:
Changed, thanks!